src/web_app_skeleton/docker/development.dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM crystallang/crystal:1.1.1
+
+# Add the nodesource ppa to apt. Update this to change the nodejs version.
+RUN wget https://deb.nodesource.com/setup_16.x -O- | bash
+
+# Apt installs:
+# - nodejs (from above ppa) is required for front-end apps.
+# - Postgres cli tools are required for lucky-cli.
+# - tmux is required for the Overmind process manager.
+RUN apt-get update && \
+    apt-get install -y nodejs postgresql-client tmux && \
+    rm -rf /var/lib/apt/lists/*
+
+# NPM global installs:
+#  - Yarn is the default package manager for the node component of a lucky
+#    browser app.
+#  - Mix is the default asset compiler.
+RUN npm install -g yarn mix
+
+# Installs overmind, not needed if nox is the process manager.
+RUN wget https://github.com/DarthSim/overmind/releases/download/v2.2.2/overmind-v2.2.2-linux-amd64.gz && \
+    gunzip overmind-v2.2.2-linux-amd64.gz && \
+    mv overmind-v2.2.2-linux-amd64 /usr/bin/overmind && \
+    chmod +x /usr/bin/overmind
```

Comment:
Should we be using [Nox](https://github.com/luckyframework/lucky_cli/blob/009f22ca4b9cdb12d807146b753a6860dc8fe91a/shard.yml#L22-L24) for this? It comes with Lucky CLI which gets installed in the next run.

Reply:
Yep. This provides overmind in the system, but if lucky-cli doesn't need it, it won't hurt anything. I needed this because I developed it against lucky-cli release rather than main.

Reply:
Ah. I forgot Nox integration hasn't been released yet. Yeah, the next release won't need a process runner!

Reply:
Since this will ship with Nox, when the cli is revised, I'm happy to pull it now or leave it in. I don't think it hurts and it might help someone get a compose going for an app that doesn't yet have the newest version of lucky on their system.